src/entities/DFGVariable.jl

```diff
@@ -130,3 +130,21 @@ internalId(v::DFGVariable) = v._internalId
 # Todo: Complete this.
 smallData(v::DFGVariable) = v.smallData
 bigData(v::DFGVariable) = v.bigData
+
+"""
+    $(SIGNATURES)
+Structure for first-class citizens of a DFGVariable.
+"""
+mutable struct DFGVariableSummary <: AbstractDFGVariable
+    label::Symbol
+    timestamp::DateTime
+    tags::Vector{Symbol}
+    estimateDict::Dict{Symbol, Dict{Symbol, VariableEstimate}}
+    _internalId::Int64
+end
```

Comment:
Big Data

Reply:
Yep, but in next PR tomorrow, get this merged tomorrow morning so we can continue, then data as another.